test/starlark_tests/targets_under_test/ios/BUILD

```
@@ -1126,6 +1144,24 @@ ios_framework(
     ],
 )
 
+ios_framework(
+    name = "fmwk_with_multiple_objc_library_and_swift_library_deps",
+    bundle_id = "com.google.example.framework",
+    families = [
+        "iphone",
+        "ipad",
+    ],
+    infoplists = [
+        "//test/starlark_tests/resources:Info.plist",
+    ],
+    minimum_os_version = "8.0",
+    tags = FIXTURE_TAGS,
+    deps = [
+        "//test/starlark_tests/resources:swift_main_lib",
```

Can you change this to `swift_shared_lib`? This ensures the case when there's more than one swift_library and one objc_library in the transitive deps is verified.

```suggestion
        "//test/starlark_tests/resources:swift_shared_lib",
```